setup.py

```diff
@@ -54,12 +51,12 @@
     packages=['tldextract'],
     include_package_data=True,
     long_description=LONG_DESCRIPTION,
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

Comment:
This is a fairly recent arg, if someone is using Ye Olde 2.6 they'd probably have old enough libraries that it would not stop them

![image](https://user-images.githubusercontent.com/2420688/53705542-40bffe80-3dda-11e9-930d-65aaf51eee3f.png)

https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires

Reply:
Fair point.

In the worst case, they have ancient pip and it will ignore this argument.

In the best case, they have modern pip and it will read the metadata and it'll automatically install the last version of this library to support 2.6.

I recommend using latest pip, setuptools (and wheel and twine) when releasing this project to make sure the argument is available to those with modern pip.